cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -316,6 +318,12 @@ class LeadImpl extends ServerImpl with Lead
       // update the Spark UI to add the dashboard and other SnappyData pages
       ToolsCallbackInit.toolsCallback.updateUI(sc)
 
+      // store every member diskStore ID to metadataCmdRgn
+      SnappyMetricsClass.putMembersDiskStoreIdInRegion()
```

Comment:
can SnappyMetricsClass.putMembersDiskStoreIdInRegion() be shifted inside SnappyMetricsClass.init(sc) method ?

Reply:
Yes, it can be shift to SnappyMetricsClass.init(sc) method.